app/contacts/[id]/page.js

```diff
@@ -0,0 +1,49 @@
+'use client';
+import { useEffect, useState } from 'react';
+
+export default function ContactDetailPage({ params }) {
+  const [contact, setContact] = useState(null);
+
+  useEffect(() => {
+    const savedContacts = JSON.parse(localStorage.getItem('contacts')) || [];
+
+    // Include default contact
+    const defaultContact = {
+      id: 123456,
+      name: 'Albert Einstein',
+      image_url: 'https://upload.wikimedia.org/wikipedia/commons/d/d3/Albert_Einstein_Head.jpg',
+      email: 'aeinstein@example.com',
+      phone_number: '15555555555',
+    };
+
+    const allContacts = [defaultContact, ...savedContacts];
+
+    const found = allContacts.find(c => c.id.toString() === params.id);
```

Comment:
beware of readability 
```suggestion
    const found = allContacts.find(contact => contact.id.toString() === params.id);
```